application/backend/app/schemas/dataset_item.py

```diff
@@ -58,6 +58,7 @@ class DatasetItemAnnotation(BaseModel):
 
     labels: list[LabelReference]
     shape: Shape
+    probability: float | None = None
```

Comment:
One more thing: could you rename this attribute "probability" to "confidence"? It's more correct because, from a mathematical perspective, this value does not represent a probability.